.cci.jenkinsfile

```diff
@@ -0,0 +1,21 @@
+// Documentation: https://github.com/coreos/coreos-ci/blob/master/README-upstream-ci.md
+
+cosaPod(buildroot: true) {
+    checkout scm
+
+    fcosBuild(make: true)
+
+    dir("/srv/fcos") {
+        stage("Build metal/live") {
+            parallel metal: {
+                shwrap("cosa build metal")
+            },
+            live: {
+                shwrap("cosa buildextend-live")
+            }
```

Comment:
We don't do any locking right now on `meta.json` though. Worse, most code paths I think read it in on start, so it's likely the last one to finish will clobber modifications.

I've thought about supporting this in cosa though. It would make the FCOS pipeline much faster. (And related here of course is https://github.com/coreos/coreos-assembler/issues/531).

Reply:
Ohh, right.